### PR TITLE
Add data transfer docs

### PR DESCRIPTION
**Pull request by masonrubenstein**

### CODE REVIEW

File: doc/data-transfer.md [hunk 1]
```diff
@@ -1,5 +1,71 @@
...
\ No newline at end of file
+**Only GCS bucket destinations are supported at this time.**
+
+## Prerequisites
+
+### USim Prerequisites
+To use USim data transfer features, one needs to make sure all USim prerequisites are met.
+USim prerequisites can be found [HERE](https://github.com/Unity-Technologies/Unity-Simulation-Docs).
+
+**[10/8/2020]** USim data transfer feature allows you to transfer data from your own project to a custom
```
Comment (qiUnity) [inline doc/data-transfer.md:13]: The date is not needed.

Comment (masonrubenstein) [reply doc/data-transfer.md]: removed

File: doc/data-transfer.md [hunk 2]
```diff
@@ -1,5 +1,71 @@
...
+**Only GCS bucket destinations are supported at this time.**
+
+## Prerequisites
+
+### USim Prerequisites
+To use USim data transfer features, one needs to make sure all USim prerequisites are met.
+USim prerequisites can be found [HERE](https://github.com/Unity-Technologies/Unity-Simulation-Docs).
+
+USim data transfer feature allows you to transfer data from your own project to a custom
+GCS bucket within region `us-central1`. If your destination is in other region, please reach out to *simulation-help@unity3d.com* and request to add your project ID to our allowlist.
```
Comment (rutvij-bhatt) [inline doc/data-transfer.md:14]: minor: Maybe rephrase as "request to add your project ID"?

**Review by qiUnity — COMMENTED** [review]
A couple of suggestions.

File: doc/data-transfer.md (hunk 2 above)
Comment (masonrubenstein) [reply doc/data-transfer.md:14]:
```suggestion
GCS bucket within region `us-central1`. If your destination is in other region, please reach out to *simulation-help@unity3d.com* and request to add your project ID. 
```

**Comment by rutvij-bhatt** [comment]
LGTM

**Review by rutvij-bhatt — APPROVED** [review]
